Cargo.toml

```diff
@@ -33,7 +33,7 @@ aes = ">=0.8"
 base64-serde = "0.6"
 base64 = "0.13"
 lazy_static = ">=1.4"
-oci-distribution = "0.7"
+oci-distribution = { git = "https://github.com/krustlet/oci-distribution" }
```

Comment:
Could we at least fix it to a specific commit, until we can get back to a tagged release?

Reply:
Yes, it make sense to me, fixed now.